src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/javadoc/JavadocLeadingAsteriskAlignCheck.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<checkstyle-metadata>
+   <module>
+      <check fully-qualified-name="com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocLeadingAsteriskAlignCheck"
+             name="JavadocLeadingAsteriskAlign"
+             parent="com.puppycrawl.tools.checkstyle.TreeWalker">
+         <description>&lt;p&gt;
+ Checks the alignment of
+ &lt;a href="https://docs.oracle.com/en/java/javase/14/docs/specs/javadoc/doc-comment-spec.html#leading-asterisks"&gt;
+ leading asterisks&lt;/a&gt; in a Javadoc comment. The Check ensures that leading asterisks
+ are aligned vertically under the second asterisk ( &amp;#42; )
+ of opening Javadoc tag. The alignment of closing Javadoc tag ( &amp;#42;/ ) is also checked.
+ If a line contains a non-whitespace character before the leading asterisk then that line will be
+ ignored. Same applies for closing Javadoc tag.
```

Comment:
`If a line contains a non-whitespace character before the leading asterisk then that line will be
 ignored.` This is kind of confusing. How can we even have a "leading asterisk" if it is not the first non-whitespace character?

Reply:
updated the description to be more specific:

https://github.com/checkstyle/checkstyle/blob/77dd1d104385ad79133495efee9f504f23f06730/src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocLeadingAsteriskAlignCheck.java#L40-L41